src/locales/en.json

```diff
@@ -67,6 +67,13 @@
       "blogLink": "Go to blog post",
       "buttonSeeMore": "See more blogs"
     },
+    "howDoesItWork": {
+      "how": "How Does It Work?",
+      "first": "Select Your Preference",
```

Comment:
the key name could be better maybe smth like preference service and connect

Reply:
Revised